pyproject.toml

```diff
@@ -6,21 +6,19 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "usercloudssdk"
-version = "1.0.5"
+version = "1.0.8"
 description = "Python SDK for UserClouds"
 authors = [{ name = "Real Python", email = "info@realpython.com" }]
 license = { file = "LICENSE" }
 dependencies = [
   "certifi >= 2023.7.2",
-  "cffi >= 1.15.1",
   "charset-normalizer >= 2.1.1",
-  "cryptography >= 41.0.2",
   "idna >= 3.4",
   "iso8601 >= 1.1.0",
   "pycparser >= 2.21",
   "PyJWT >= 2.6.0",
   "requests >= 2.31.0",
-  "urllib3 >= 1.26.12"
+  "urllib3 >= 1.26.6"
```

Comment:
when i run the following (for aws lambda environment):
```
pip install     --platform manylinux2010_x86_64     --implementation cp         --only-binary=:all: --upgrade     --target awsbundle2     usercloudssdk==1.0.8
```

there was no available urllib3 on manylinux2010_x86_64 with version 1.26.12 or higher

Reply:
maybe use a newer version, non-legacy tag: https://peps.python.org/pep-0600/#core-definition
`manylinux2010_x86_64` is very old....
we shouldn't be using it.

Reply:
hmm i think i started using the 2010 platform because of this thread:
https://github.com/pyca/cryptography/issues/6390

but I do see the current AWS lambda documentation saying to use the 2014 platform:
https://docs.aws.amazon.com/lambda/latest/dg/python-package.html

I'll try to use 2014 in my next deploy and see if I can revert this urllib3 change

Reply:
python 3.7 is pretty old and AWS Lambda supports newer python versions.
so I think it is ok for us to say we only support python 3.8 and above.
![Screenshot 2023-09-05 at 12 45 57 PM](https://github.com/userclouds/sdk-python/assets/1268088/cc7c60df-67ee-4c87-ab99-49568f6bcd4f)

Reply:
It appears the urllib3 change was unnecessary so I will revert it! There were some other concurrent moving pieces when I thought it was necessary (e.g. I did change the lambda to use Python 3.11 as well)